### DISCUSSION TITLE
Can not open port for pfsense integration

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#12210] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

32C/64T

### RAM

124 GB

### Storage for /

314 GB

### Storage for /nsm

1271 GB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello all

I have followed the documentation for the [Elastic Integration for pfSense](https://docs.securityonion.net/en/2.4/pfsense.html#elastic-integration-for-pfsense) for a opnsense firewall/router.

The opnsense is configured for udp forwarding on port 9001 and states in his log that the connection is established and then broken.
![Screenshot from 2024-01-19 12-35-51](https://github.com/Security-Onion-Solutions/securityonion/assets/29328171/60a3a2fb-ab95-4d9e-94be-911f13b112cb)

If I do a nmap scan the port is not marked as open.

![Screenshot from 2024-01-19 10-49-18](https://github.com/Security-Onion-Solutions/securityonion/assets/29328171/ebbf31e9-2541-444a-94a2-0587f2addf37)

with the so-firewall command you can not check all the open ports.

I did a iptables grep and this is the result
`  230 55285 ACCEPT     udp  --  *      *       x.x.x.x         0.0.0.0/0            udp dpt:9001
`
But the kibana dashboards for this integration stays empty.

What could be cause of this?

For being complete currently we receive the log over syslog (port 514).

regards
Bart

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TotieBash** [comment]
For what its worth I also has similar issue #12055.  When you issue cmd "docker ps" one of those docker container should show that it is listening on port 9001.  I think it should be so-logstash.  Also when you issue cmd "ss -tulpn" you should see the system listening on port 9001.  However I don't know the missing step to get to that.  I have done it on SO2.3 Filebeat method but this SO2.4 method is new to me.

**Comment by TotieBash** [comment, marked as answer]
I figured out my issue, check #12055

I was incorrect on my assumption that so-logstash should be listening on port 9001.  Apparently there is "elastic-agent" that is managed by fleeet installed on the standalone host itself so it is not a docker container.  You can verify this when you issue "systemctl status elastic-agent"
